test/index.test.js

```diff
@@ -866,5 +866,27 @@ describe('mongoose module:', function() {
       const m = new mongoose.Mongoose();
       assert.deepEqual(await m.syncIndexes(), {});
     });
+    it('Allows a syncIndexes global option (gh-11030)', async function() {
+      const m = new mongoose.Mongoose();
+
+      const db = await m.connect('mongodb://localhost:27017/mongoose_test_11030');
+      const schema = new m.Schema({
+        title: { type: String, index: true }
+      });
+      const syncFalse = db.model('Sync', schema, 'Sync');
+      await syncFalse.collection.dropIndexes();
```

Comment:
I think you need to get rid of this line to fix the tests. The problem is that the 'Sync' collection doesn't exist yet when you're calling `dropIndexes()` here.